docker/python-dev/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8-alpine
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev python3-dev 
+RUN apk add --no-cache bash make
+
+COPY requirements.txt /tmp/
+RUN pip install -r /tmp/requirements.txt
+
+RUN apk del --no-cache .build-deps
+
+RUN mkdir -p /src
```

Comment:
Put `src` and `tests` under `/workspace` might be better? Otherwise people may be thought that the files under `/src` are system files? After all, `WORKDIR /workspace` might be more clear for users.

Reply:
Yes, be consistent with `golang`

But I think we need highlight the definition of `workspace` folder, this is a folder created by us.